src/Bridge/Symfony/Bundle/DependencyInjection/ApiPlatformExtension.php

```diff
@@ -262,7 +263,8 @@ private function registerMetadataConfiguration(ContainerBuilder $container, arra
 
         if (!empty($config['resource_class_directories'])) {
             $container->setParameter('api_platform.resource_class_directories', array_merge(
-                $config['resource_class_directories'], $container->getParameter('api_platform.resource_class_directories')
+                $config['resource_class_directories'],
```

Comment:
We should avoid to change the code style on `main` (we can do it in `2.6`): conflicts will be harder to solve when merging `2.6` into `main`.